schutzbot/Jenkinsfile

```diff
@@ -20,6 +20,8 @@ pipeline {
         AZURE_SUBSCRIPTION_ID = "8d026bb1-2a65-454d-a88f-c896db94c4f8"
         AZURE_RESOURCE_GROUP = "sharing-research"
         AZURE_LOCATION = "eastus"
+        API_TEST_SUBSCRIPTION_ORG_ID = credentials('api-test-subscription-org-id')
+        API_TEST_SUBSCRIPTION_KEY = credentials('api-test-subscription-activation-key')
```

Comment:
Wait, aren't we supposed to be testing subscriptions for image-builder ? Why is this change in the osbuild-composer repository then ? I don't follow ?

Reply:
Hi @atodorov ,
I don't think we have case in image-builder to verify the images. But perhaps I misunderstood something. I'll double check with Sanne about this test. Thanks!

Reply:
I might have caused some confusion here yesterday. I think it's fine to add it in both, it would essentially be testing the same thing. But since the plan was Image builder let's start there.

Reply:
Hi @atodorov ,
Sorry, my fault. I Just double confirm with Sanne and I'll move it to image-builder project. Thanks!